frontend/src/_layouts/base.html

```diff
@@ -101,6 +96,15 @@
       </div>
     </header>
 
+    <!-- Top section subscribe now -->
+    <section class="hidden flex-wrap items-center justify-center gap-2 border-gray-300 bg-neutral-100 px-4 py-2 sm:flex-nowrap">
+      {% render 'partials/subscribe.html' -%}
+    </section>
+    <!-- Top section transcribe now -->
+    <section class="flex flex-wrap items-center justify-center gap-2 border-b border-gray-300 bg-neutral-100 px-4 py-2 sm:flex-nowrap">
+      {% render 'partials/transcribe.html' -%}
+    </section>
```

Comment:
I believe that the surrounding tags should be moved inside partials because it's not sustainable to have a `flex` container outside a partial. The layout logic is spread in this case.

Also, we can completely omit the subscription partial.